runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -81,12 +92,56 @@
 
 	private static final Set<Integer> BARROWS_LADDERS = Sets.newHashSet(NullObjectID.NULL_20675, NullObjectID.NULL_20676, NullObjectID.NULL_20677);
 
+	private static final int TUNNEL_WIDGET_GROUP_ID = 229;
```

Comment:
I've currently placed this here because I'm unsure how to exactly add these to `WidgetID` and `WidgetInfo` in case that's the preferred method.

Reply:
It is not preferred method, it is required method, simply add new entry to WidgetId (i think there is already "group" for barrows) and then use it in WidgetInfo